stdlib/public/core/CollectionAlgorithms.swift.gyb

```diff
@@ -269,6 +269,9 @@ extension ${Self} {
   /// Returns the elements of the ${sequenceKind}, sorted using the given
   /// predicate as the comparison between elements.
   ///
+  /// This method can take throwing closure. If closure throws error while 
+  /// sorting, it will return empty collection.
```

Comment:
This seems like the wrong behavior.  Why isn't the error simply propagated outward (in which case nothing is actually returned)?